### PR TITLE
Issues 86 - Add basic personal skill

### PR DESCRIPTION
**Pull request by ethanaward**
This adds a basic skill to ask Mycroft personal details about himself.

### CODE REVIEW

**Comment by aatchison** [comment]
Cool, need a review assignee?

**Comment by ryanleesipes** [comment]
Would like Mycroft to say (to "what are you?"), "I am an open source artificial intelligence."

**Comment by ryanleesipes** [comment]
Assigned to @Wolfgange3311999 - please review.

**Comment by jdorleans** [comment]
@ethanaward I'd like to have one more option for the questions and responses. Do you mind to add one more?

@ryanleesipes could you review the answers and check if they are reliable?

After that we can merge! : D

**Comment by ethanaward** [comment]
@jdorleans: Sure, what response do you have in mind?

**Comment by Wolfgange3311999** [comment]
How about a few alternative responses in the dialog file? So, for who are you maybe "I am an open source AI", "I'm an intelligent piece of software for communicating with machines", "I am evil"

Okay, maybe not the last one but a few for variety. Other than that it looks good.

File: mycroft/skills/personal/dialog/en-us/who.am.i.dialog [hunk 1]
```diff
@@ -0,0 +1,2 @@
+I'm Mycroft. Didn't you know that?
```
Comment (jdorleans) [inline mycroft/skills/personal/dialog/en-us/who.am.i.dialog]: Does this sound a bit rude? hahah... What about:
"My name is Mycroft and I'm an intelligent personal assistant"

**Comment by aatchison** [comment]
What's the status of this PR?

**Comment by ethanaward** [comment]
I've made all of @jdorleans' suggested changes, all that's left is possibly for @ryanleesipes to check over the responses.

**Comment by ryanleesipes** [comment]
This LGTM.

**Pull request merged**